### PR TITLE
Fast lane action workers

### PR DESCRIPTION
**Pull request by kvahed**
fixes https://github.com/arangodb/team-clifton-planning/issues/8

### CODE REVIEW

File: arangod/Cluster/MaintenanceFeature.cpp [hunk 1]
```diff
@@ -62,6 +62,7 @@ void MaintenanceFeature::init() {
   requiresElevatedPrivileges(false); // ??? this mean admin priv?
 
   // these parameters might be updated by config and/or command line options
+
   _maintenanceThreadsMax = (std::max)(static_cast<uint32_t>(2),
```
Comment (lamai93) [inline arangod/Cluster/MaintenanceFeature.cpp]: Please set this to 3, otherwise we might get https://github.com/arangodb/arangodb/pull/6310 again.

**Comment by neunhoef** [comment]
This is tracked by https://github.com/arangodb/team-clifton-planning/issues/8

File: arangod/Cluster/MaintenanceFeature.cpp [hunk 2]
```diff
@@ -39,6 +39,9 @@ using namespace arangodb::application_features;
 using namespace arangodb::options;
 using namespace arangodb::maintenance;
 
+const uint32_t MaintenanceFeature::minThreadLimit = 2;
```
Comment (matthewvon) [inline arangod/Cluster/MaintenanceFeature.cpp]: This has to be 3 to pass Jenkins.  There was another PR that discussed the issue and a comment on the PR saying same.

Comment (matthewvon) [reply arangod/Cluster/MaintenanceFeature.cpp]: Should also add a comment so someone like me does not come back later and screw this up.

**Pull request merged**